easybuild/easyconfigs/p/prereq/prereq-tf-2.2-CrayGNU-19.10.eb

```
@@ -0,0 +1,60 @@
+# contributed by sarafael (CSCS)
+easyblock = 'PythonBundle'
+
+name = 'prereq'
+version = 'tf-2.2'
+_pyminver = '6'
+
+homepage = 'https://pypi.python.org/pypi'
+description = """This module is a bundle of Python packages on Cray systems based on the module cray-python"""
+
+toolchain = {'name': 'CrayGNU', 'version': '19.10'}
+toolchainopts = {'pic': True, 'verbose': False}
+
+dependencies = [
+    ('cray-python/3.6.5.7', EXTERNAL_MODULE),
+    ('numpy', '1.17.2'),
+    ('h5py', '2.8.0', '-python%(pymajver)s-serial'),
+    ('PyExtensions', '%(pyver)s'),
+]
+
+exts_default_options = {
+    'req_py_majver': '%(pymajver)s',
+    'req_py_minver': '%s' % _pyminver,
+    'source_urls': [PYPI_SOURCE]
+}
+
+exts_list = [
+    ('backports.weakref', '1.0.post1'),
+    ('Werkzeug', '0.16.0'),
+    ('absl-py', '0.8.0', {
+        'modulename': 'absl',
+    }),
+    ('Keras_Applications', '1.0.8'),
+    ('Keras_Preprocessing', '1.1.0'),
+    ('gast', '0.3.3'),
+    ('astor', '0.8.0'),
+    ('termcolor', '1.1.0'),
+    ('chardet', '3.0.4'),
+    ('certifi', '2019.9.11'),
+    ('idna', '2.8'),
+    ('urllib3', '1.25.6'),
+    ('requests', '2.22.0'),
+    ('wrapt', '1.11.2'),
+    ('setuptools', '41.2.0', {
+        'source_tmpl': 'setuptools-41.0.1.zip',
```

```suggestion
        'source_tmpl': 'setuptools-%(version)s.zip',
```
I think this would work as well